### PR TITLE
- fixed duplicate volume group name - ported on OL8

### PR DESCRIPTION
**Pull request by guidotijskens**
fixed the issue of the danger for duplicate volume group names,
updated code and spec for execution on OL8/UEK6

tests:
 - ran a set of tests verifying and modifying qcow2 and vmdk images, with and without logical volumes, on OL7 and OL8

### CODE REVIEW

File: lib/oci_utils/migrate/imgdevice.py [hunk 1]
```diff
@@ -773,11 +773,12 @@ def collect_os_data(self):
             #
             # import operation system type dependant modules
             osrelease = self.get_os_release()
```
Comment (ejannett) [inline lib/oci_utils/migrate/imgdevice.py:775]: IT seams that method return a dict , why such cast ?

Comment (guidotijskens) [reply lib/oci_utils/migrate/imgdevice.py:775]: just want to collect the full os release data, and save it in a central repo; currently only need the 'ID', i.e. the flavor of the distro, but extension are in the pipeline, we do not need to look for any release related data again, even if the filesystems, logical volumes, imagefiles are not mounted anymore.

Comment (ejannett) [reply lib/oci_utils/migrate/imgdevice.py:775]: Sorry . the spotted line is wrong . My question is about the statement
" bool(osrelease) "

Comment (guidotijskens) [reply lib/oci_utils/migrate/imgdevice.py:775]: just testing if the dictionary is empy, bool() calls the __bool__ ( or in fact __len__) method;
could have been len() > 0 or so, but thought bool() is more clear ....

**Comment by guidotijskens** [comment]
obsolete

**Pull request closed without merging**